Intersect.Client.Framework/Gwen/Input/InputHandler.cs

```diff
@@ -395,13 +396,41 @@ public static bool OnMouseClicked(Base canvas, int mouseButton, bool down)
         }
 
         /// <summary>
-        ///     Key handler.
+        ///     Mouse click handler.
         /// </summary>
         /// <param name="canvas">Canvas.</param>
-        /// <param name="key">Key.</param>
-        /// <param name="down">True if the key is down.</param>
+        /// <param name="mouseButton">Mouse button number.</param>
+        /// <param name="down">Specifies if the button is down.</param>
         /// <returns>True if handled.</returns>
-        public static bool OnKeyEvent(Base canvas, Key key, bool down)
+        public static bool OnMouseScroll(Base canvas, int deltaX, int deltaY)
+        {
+
+            if (canvas == null)
+                return false;
```

Comment:
Can you make all of these `{}`d for consistency's sake

Reply:
Sure thing, condensed them as well.